game/cards/dm06/guardian.go

```diff
@@ -0,0 +1,54 @@
+package dm06
+
+import (
+	"duel-masters/game/civ"
+	"duel-masters/game/cnd"
+	"duel-masters/game/family"
+	"duel-masters/game/fx"
+	"duel-masters/game/match"
+)
+
+func LuGilaSilverRiftGuardian(c *match.Card) {
+
+	c.Name = "Lu Gila, Silver Rift Guardian"
+	c.Power = 4000
+	c.Civ = civ.Light
+	c.Family = []string{family.Guardian}
+	c.ManaCost = 5
+	c.ManaRequirement = []string{civ.Light}
+
+	c.Use(fx.Creature, fx.Blocker, fx.CantAttackPlayers, func(card *match.Card, ctx *match.Context) {
+
+		if card.Zone != match.BATTLEZONE && card.Zone != match.HIDDENZONE {
```

Comment:
I don't think the HIDDENZONE check is needed here (?) If you put another evolution card on this one (stack of 3) only the top card's effects should apply as far as I know

Reply:
This is needed for the case where Lu GIla was the card that just evolved. According to the rulings (https://duelmasters.fandom.com/wiki/Lu_Gila,_Silver_Rift_Guardian/Rulings) if you evolve Lu Gila, it's evolution also becomes tapped.